modules/ROOT/pages/amazon/amazon-s3-connector-design-center.adoc

```diff
@@ -4,42 +4,58 @@ include::_attributes.adoc[]
 endif::[]
 :imagesdir: ../../assets/images/
 
-To configure a connector in Design Center, follow this process:
+Design Center enables you to create apps visually. To use Design Center, work with your Anypoint Platform administrator to ensure that you have a xref:access-management::environments.adoc#to-create-a-new-environment[Design environment]. For more information, see the xref:design-center::fd-tour.adoc[Flow Designer Tour].
 
-. Configure the trigger for the connector, which can be an HTTP Listener, a Scheduler, or an operation from the connector (if available).
+To create an app in Design Center:
 
-. Configure the target component for the connector.
+* Configure the input source (trigger) for your app.
+* Add the connector as a component to process the input for the app.
 
-== Configure the Amazon S3 Connector in Design Center
+For information about Amazon S3 fields, see the xref:amazon/amazon_S3_connector-reference[Amazon S3 Connector Reference].
 
-To configure the Amazon S3 connector in Design Center:
+== Configure the Trigger
 
-. From the Anypoint Platform Design Center, click *Create*, and then click *Create new application* under *Create a Mule Application*.
-. Enter a project name and click *Create*.
-. Exit the *Let’s Get Started* wizard by clicking *Go straight to canvas*.
-. Click the plus sign.
-. Select a trigger. To have an Amazon S3 event initiate access to your app (that is, to trigger your app), select the Amazon S3 Connector and then select one of the Amazon S3 trigger operations. Alternatively, select an HTTP Listener or Scheduler as the trigger.
+To configure a trigger:
 
+. In Design Center, click *Create*.
+. Click *Create new application*.
+. Specify a value for *Project name*, and click *Create*.
+. Exit from *Let's get started* by clicking *Go straight to canvas*.
+. Click on the Trigger card.
+. If you are using your connector as an input source trigger, search for the connector and select an operation. Otherwise, search for and select *HTTP* or *Scheduler*.
```

Comment:
"your connector" is incorrect. The connector is not something a user owns, but merely uses. How about "this connector" ?